provider/test/emptycollection/provider.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 
 	"github.com/go-spatial/geom"
-	"github.com/go-spatial/tegola/proj"
+	"github.com/go-spatial/proj"
```

Comment:
I know it's functionally not different, but please revert this import to the `tegola/proj` package so we're consistent.